frontend/app/components/dropdown/dropdown.tsx

```diff
@@ -1,10 +1,13 @@
 /** @jsx createElement */
 import { createElement, Component, createRef } from 'preact';
 import b from 'bem-react-helper';
+import nodeEmoji from 'node-emoji';
```

Comment:
It looks strange, that common component knows about emojis. Could you invert this dependency?